README.md

```diff
@@ -46,11 +53,36 @@ For example
 ### Run
 
 ```
-./start-all.sh
+./start-all.sh -d data_dir
 ```
 
-### Load original data to prometheus server
+<table style="border-style: solid;">
+  <tr>
+    <td>
+ ** Note: The -d data_dir is optional, but without it, prometheus will erase all data between runs.
+ <br>
+ For systems in production it is recomended to use an external directory. **
+</td>
+</tr>
+</table>
+
+### Kill
+
+```
+./kill-all.sh
+```
+
+### Use
+Direct your browser to `your-server-ip:3000`
+
+#### Choose Disk and network interface
+The dashboard holds a drop down menue at its upper left corner for disk and network interface.
+You should choose relevent disk and interface for the dashboard to show the graphs. 
+
```

Comment:
I agree, I'll add that as a separate patch after the general README will be merged